src/Backend/Modules/Blog/Layout/Templates/EditCategory.html.twig

```diff
@@ -63,7 +63,7 @@
           </button>
         </div>
       </div>
-      {% if showBlogDeleteCategory %}
+      {% if showBlogDeleteCategory and allowBlogDeleteCategory %}
```

Comment:
@carakas This is something different.
See: https://github.com/forkcms/forkcms/pull/1828/files#diff-8f7f5362034515c7d4353de51b90b63aR83